src/Native/pkg/runtime.native.System.IO.Compression/ubuntu/14.04/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -1,19 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  
   <PropertyGroup>
-    <Version>1.0.1</Version>
+    <Version>4.1.1</Version>
```

Comment:
For all split packages I made sure that the runtime packages had the same version as the identity packages.  I also have a change that validates this in buildtools but didn't want to check that into buildtools yet as it would be dependent on this PR being merged.